astroid/brain/brain_typing.py

```diff
@@ -52,6 +50,11 @@ def __args__(self):
 class {0}(metaclass=Meta):
     pass
 """
+# PEP484 suggests NewType is equivalent to this for typing purposes
```

Comment:
Could you add a link to section in the PEP? Saves a potential contributor from having to search for it.